modules/fscloud/README.md

```diff
@@ -1,13 +1,13 @@
 # Pre-wired CBR configuration for FS Cloud
 
 This module creates default coarse-grained CBR rules in a given account following a "secure by default" approach - that is: deny all flows by default, except known documented communication in the [Financial Services Cloud Reference Architecture](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-vpc-architecture-about):
-- COS -> KMS
-- Block storage -> KMS
-- ROKS -> KMS
+- COS -> KMS and HPCS
```

Comment:
I would just go with HPCS, with 1 or 2 lines of text explaining how to use key protect instead

Reply:
Done.

Reply:
Done.